horde/model_reference.py

```diff
@@ -40,11 +40,11 @@ def call_function(self):
                 if self.text_reference[model].get("nsfw"):
                     self.nsfw_models.add(model)
 
-        except Exception:
+        except Exception as e:
             logger.error(f"Error when downloading known models list: {e}")
 
     def get_model_names(self):
-        return set(reference.keys())
+        return set(self.reference.keys())

```

Comment:
@db0 is this related to the model issues you're having?

Reply:
No that was a caching issue

Reply:
I don't actuallyt use this method afaics